strangerstudios/paid-memberships-pro
# Fixed filtered PayPal order description not being sent.

### All Submissions:






### Changes proposed in this Pull Request:



Fixed issues where some order descriptions sent to PayPal would not be filtered using `pmpro_paypal_level_description`. This could cause failed checkouts in some cases, particularly with subscription memberships using PayPal Express.

### How to test the changes in this Pull Request:

1. Set up PayPal Express as gateway
2. Create membership level with recurring payment
3. Install this Gist which prepends the user's username to the PayPal description: https://gist.github.com/dparker1005/f048542f620038d91bc96f9baa4d9cc5
4. Check out for a membership
5. View the order details in PayPal and verify that the order description has the user's username

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
* BUG FIX: Fixed issue where pmpro_paypal_level_description filter could sometimes caused failed checkouts
